wasabi/research-club/anonymous-credentials.md

```diff
@@ -1,695 +1,327 @@
 
-Speaker 1: 00:03:20
+Unknown speaker  5:Unknown speaker6  
+Thank you. Did you finish?
 
-Stefano, I think, not Peter or something like that.
-Okay.
-
-Speaker 0: 00:03:24
-
-He wrote his PhD thesis.
-Okay, Branz is his last name.
-And he wrote a PhD thesis about his idea of doing credentials.
-And it's actually quite interesting.
-I can recommend it.
-It's not too technical.
-And he has a lot more ideas than just credentials.
-But one problem with it was that so far, every attempt at proving it secure in the random Oracle model failed.
-And this is where this anonymous credentials light paper comes in because they have a construction that is provably secure And you can do basically the same things or very similar things as with brands credentials.
-There might be one minor problem.
-Last time I looked at it, I did not really solve because in the paper they say for 128-bit security, we recommend using a group of 576 bits.
-And Of course, our SecP group is only 256 bits.
-So it's not sure if we can use that.
-It's often the case that due to the proof, these groups need to be much larger than they are used in practice.
-Even for Schnorr signatures, our SecP group would actually be too small for 128-bit security.
-Mostly people say, yeah, that's just an artifact of the proof, it doesn't really matter that much, but in order to verify that, we would need to look into the proof more closely.
-
-Speaker 1: 00:05:24
-
-Yeah.
-Thank you.
-Did you finish?
-
-Speaker 0: 00:05:31
-
-Yeah, that's almost everything I wanted to say.
-Like the other thing I wanted to say is that these tokens are pretty flexible compared to just Schnorr signatures.
-So at the Building on Bitcoin Conference 2018 I talked about how to use this to basically swap a token that is in or that it was created with an anonymous credential with an on-chain or off-chain Bitcoin on Lightning without the server knowing that this happened and without having to trust the other party.
+Jonas  5:31  
+Yeah, that's almost everything I wanted to say. Like, the other thing I wanted to say is that these tokens are pretty flexible compared to just Schnorr signatures. So at the building on Bitcoin Conference, Unknown speaker018, I talked about how to use this to basically swap a token that is in or that is most created with an anonymous credential with an on chain or off chain Bitcoin on lightning without the server knowing that this happened and without having to trust the other party.
```

Comment:
There is a random "Unknown speaker" text inside of this text

Reply:
My bad I put that there while I was editing.